.github/scripts/wasm-target-test-build.sh

```diff
@@ -0,0 +1,17 @@
+GIT_ROOT=$(pwd)
```

Comment:
I would suggest adding this line at the beginning of the file so that the OS can detect it as a shell script:

```suggestion
#!/bin/sh

GIT_ROOT=$(pwd)
```